### ISSUE TITLE
People don't realize they can resize sublists

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#14542]

**Comment by Thatoo** [comment]
Indeed, I didn't realize that before I read your issue. The small line that appear to let us resize should be visible all the time, maybe not as dark as when you hover it but at least it should be there (same color as the sentence "show x more").

**Comment by niquewoodhouse** [comment]
I think before the background gradients and blur were introduced, this was a bit easier to spot and operated fairly similarly to resize bars in things like Gmail.

![gmail-resize](https://user-images.githubusercontent.com/1587166/89874457-c4774080-dbb3-11ea-99bf-bb5d18c7dd9a.gif)

There's a lot more UI now for the eye - @Thatoo's idea, or some variant of it, sounds pretty good imo.

**Comment by t3chguy** [comment]
Fixed by new room list
